lib/grunt/utils.js

```diff
@@ -6,11 +6,39 @@ var shell = require('shelljs');
 var grunt = require('grunt');
 var spawn = require('child_process').spawn;
 var semver = require('semver');
+var rimraf = require('rimraf');
 
 var _ = require('lodash');
 
 var CSP_CSS_HEADER = '/* Include this file in your html if you are using the CSP mode. */\n\n';
 
+var SHRINKWRAP_FILE = 'npm-shrinkwrap.json';
+var SHRINKWRAP_CACHED_FILE = 'node_modules/npm-shrinkwrap.cached.json';
+
+function shrinkwrapChanged() {
+  return shell.exec('diff ' + SHRINKWRAP_FILE + ' ' + SHRINKWRAP_CACHED_FILE, {silent: true}).output;
+}
+
+function cleanNodeModules() {
+  if (process.platform == "win32") {
+    rimraf.sync('node_modules');
+    console.log('cleaned node_modules using rimraf');
+  } else {
```

Comment:
I think rm -rf might still be more robust, so we should use it if possible. @IgorMinar should probably have a word here.